apps/mobile/ios/KlaytnWallet.xcodeproj/project.pbxproj

```diff
@@ -690,6 +693,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				INFOPLIST_KEY_CFBundleDisplayName = "";
```

Comment:
take a look at this file, looks like it have redundant changes. for example `TEST_HOST`, `IPHONEOS_DEPLOYMENT_TARGET` and `INFOPLIST_KEY_CFBundleDisplayName` sometimes empty